src/unity/python/turicreate/toolkits/drawing_recognition/drawing_recognition.py

```diff
@@ -0,0 +1,483 @@
+import turicreate as _tc
+import urllib as _urllib
+import os as _os
+import glob as _glob
+import numpy as _np
+import mxnet as _mx
```

Comment:
Hmm, I do need all the mxnet imports because I have a Model class that inherits HybridBlock. Should I defer all the other mxnet imports to where I need them?

Reply:
I think you're probably going to need to move that model class to a separate file, and not import that file until it's needed.

Reply:
Thanks for the suggestion, fixed now.